ransack.gemspec

```diff
@@ -16,11 +16,11 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activerecord', '>= 5.2.1'
   s.add_dependency 'activesupport', '>= 5.2.1'
+  s.add_dependency 'machinist', '~> 1.0.6'
```

Comment:
Why is this runtime dependency being added? :thinking:

Reply:
Thanks for the review @deivid-rodriguez !

Has now been removed.